README.md

```diff
@@ -22,6 +22,12 @@ if you want to use the BGE-M3 encoder with `pyterrier_dr`, you can install the p
 pip install pyterrier-dr[bgem3]
 ```
 
+If you want to use `lightning-ir` models with `pyterrier_dr`, you can install the package with the `lightning-ir` dependency:
```

Comment:
add hyperlink to lightning IR so I can find out more about what they are about? anything special about them apart from a name?

Reply:
fine - it just needs to adds some links